### ISSUE TITLE
Manage incomplete data in SRT

### ISSUE THREAD

**Issue by tatoz12** [JuanIrache/DJI_SRT_Parser#8]
Sometimes the DJI drones write incomplete information in the SRT, like this:

```
1
00:00:01,000 --> 00:00:02,000
HOME(-58.3718,-38.4799) 2019.08.08 17:43:03
GPS(-58.3592,-38.4693,16) BAROMETER:122.8
ISO:100 Shutter:160 EV: 0 Fnum:F2.8 

2

-- the problem--
```

Right now, when you upload incomplete files, the app throws error. So, the solution i found is to add this line of code: `arr = arr.filter(value => Object.keys(value).length !== 0);` at the begining of `DJI_SRT_Parser.prototype.interpretMetadata` function, to clean emtpy/incomplete values in the array.

Sorry for my english.

**Comment by JuanIrache** [comment]
Hi, nice catch.

Could you make the change and send a pull request? Otherwise I'll do it when I get a moment.

Thanks

**Comment by tatoz12** [comment]
I just did it, but I don't know if I did it well (first time with git). 
By the way, thanks for the amazing apps!

**Comment by JuanIrache** [comment]
I think you sent the pull request to your own fork only. You should send it to mine. I'll have a look into it later if you can't solve it.

**Comment by JuanIrache** [comment]
Fixed in 049a25ec293763b55f5537cc3c1f7d7e1739d2d1

Thanks
